health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L2Main.js

```
@@ -0,0 +1,757 @@
+    }),
+    [filters, isUlbLoading, isServicesLoading]
+  );
+
+  const mobileView = window.Digit.Utils.browser.isMobile();
+
+  const handlePrint = () => Digit.Download.PDF(fullPageRef, t(dashboardConfig?.[0]?.name));
+
+  const removeULB = (id) => {
+    handleFilters({
+      ...filters,
+      filters: { ...filters?.filters, tenantId: [...filters?.filters?.tenantId].filter((tenant, index) => index !== id) },
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)